features/mbedtls/targets/TARGET_STM/aes_alt.c

```diff
@@ -304,26 +389,41 @@ int mbedtls_aes_crypt_ctr( mbedtls_aes_context *ctx,
 }
 #endif /* MBEDTLS_CIPHER_MODE_CTR */
 
-void mbedtls_aes_encrypt( mbedtls_aes_context *ctx,
+int mbedtls_internal_aes_encrypt( mbedtls_aes_context *ctx,
                           const unsigned char input[16],
                           unsigned char output[16] )
 {
-    
     if (HAL_CRYP_AESECB_Encrypt(&ctx->hcryp_aes, (uint8_t *)input, 16, (uint8_t *)output, 10) !=0) {
-        // error found to be returned
+        // error found
+        return 1;
```

Comment:
Normally mbed TLS errors are negative integers no larger than 2 bytes. What should the hardware acceleration error codes look like? @RonEld @yanesca